pkg/sql/compile/ddl.go

```diff
@@ -48,7 +48,7 @@ func (s *Scope) DropDatabase(c *Compile) error {
 		if s.Plan.GetDdl().GetDropDatabase().GetIfExists() {
 			return nil
 		}
-		return err
+		return moerr.NewErrDropNonExistsDB(dbName)
 	}
 	return c.e.Delete(c.ctx, dbName, c.proc.TxnOperator)
```

Comment:
Line 42, we called CreateAutoIncrTable.   Should we clean it up in drop?

Reply:
No, deleting the database will delete all the tables, including the tables needed for auto increment columns